js/src/components/contact-information/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useGoogleMCPhoneNumber from '.~/hooks/useGoogleMCPhoneNumber';
+import Section from '.~/wcdl/section';
+import VerticalGapLayout from '.~/components/vertical-gap-layout';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import AppSpinner from '.~/components/app-spinner';
+import PhoneNumberCard from './phone-number-card';
+
+const description = __(
+	'Your contact information is required by Google for verification purposes. It will be shared with the Google Merchant Center and will not be displayed to customers.',
+	'google-listings-and-ads'
+);
+
+const mcTitle = __( 'Enter contact information', 'google-listings-and-ads' );
+const settingsTitle = __( 'Contact information', 'google-listings-and-ads' );
+
+export default function ContactInformation( { view, onPhoneNumberChange } ) {
+	const phone = useGoogleMCPhoneNumber();
+	const isSetupMC = view === 'setup-mc';
+
+	const initEditing = isSetupMC ? ! phone.data.isValid : true;
+	const title = isSetupMC ? mcTitle : settingsTitle;
+	const trackContext = isSetupMC
+		? 'setup-mc-contact-information'
+		: 'settings-contact-information';
+
+	return (
+		<Section
+			title={ title }
+			description={
+				<div>
+					<p>{ description }</p>
+					<p>
+						<AppDocumentationLink
+							context={ trackContext }
+							linkId="contact-information-read-more"
+							// TODO: [lite-contact-info] add link
+							href="https://example.com/"
+						>
+							{ __( 'Learn more', 'google-listings-and-ads' ) }
+						</AppDocumentationLink>
+					</p>
+				</div>
+			}
+		>
+			{ phone.loaded ? (
+				<VerticalGapLayout size="large">
+					<PhoneNumberCard
+						phoneNumber={ phone }
+						initEditing={ initEditing }
+						onPhoneNumberChange={ onPhoneNumberChange }
+					/>
+					<div>TODO: add store address card</div>
+				</VerticalGapLayout>
+			) : (
+				<AppSpinner />
+			) }
```

Comment:
Speaking of https://github.com/woocommerce/google-listings-and-ads/pull/917#discussion_r681156476 I think this loading state handling is redundant, and delivers slightly worse UX.

We can remove the redundancy from `ContactInformation` and let the `PhoneNumberCard` handle loading state itself.
```suggestion
			<VerticalGapLayout size="large">
				<PhoneNumberCard
					phoneNumber={ phone }
					initEditing={ initEditing }
					onPhoneNumberChange={ onPhoneNumberChange }
				/>
				<div>TODO: add store address card</div>
			</VerticalGapLayout>
```

Reply:
After the change suggested above, we would have to change `initEditing` handling.
Personally, in `ContactInformation` I'd cater for `isSetupMC` only. Then let `PhoneNumberCard` reason about `phone.data.isValid` + some given property.
.

Reply:
Compare this way to https://github.com/woocommerce/google-listings-and-ads/pull/917#discussion_r681653834, it could only deliver the phone icon and "Phone number" title earlier about 0.5 ~ 1.5 seconds at the most. The rest elements are still needed to wait for data loaded. I'm not sure if that is worth it.

Reply:
I'd say 0.5~1.5 s on a developer machine is a lot. Then imagine how long it could take on a low-end device with a slow network.

> I'm not sure if that is worth it.

Personally, I see it the opposite, compare:
- Deliver a scaffold instantly, with less code, and a single component that's responsible for the preloading state of its data
- Deliver a blank space (or card), with more code, and preloading logic and responsibility scattered around 2 components (3 in #921)

Reply:
Opened another PR #934 to change the handling of loading states.